src/main/scala/rps/Main.scala

```diff
@@ -59,4 +54,16 @@ object WebServer extends App with RouterDerivationModule {
   object Draw
 }
 
-case class Response (userMove: Move, computerMove: Move, result: Result)
\ No newline at end of file
+case class Response (userMove: Move, computerMove: Move, result: Result)
+
+trait ID {
+  type IdType
+  def generateGameId (game : Response) :IdType
+}
+
+object HashId extends ID {
+  type IdType = Int
+  def generateGameId (game: Response) : IdType = {
+      game.##
```

Comment:
Hashes can collide, I wouldn't use them as an Id! 
You can use something like an `AtomicInteger` and just increment it every time, or a `UUID`